examples/gallery/ui/pages/controls_page.slint

```diff
@@ -15,50 +15,57 @@ export component ControlsPage inherits Page {
         title: "Buttons";
 
         HorizontalLayout {
-            spacing: 8px;
             alignment: start;
+            spacing: 8px;
 
             Button {
                 text: "Regular Button";
-                enabled: !GallerySettings.widgets-disabled;
+                enabled: GallerySettings.widgets-enabled;
             }
 
             Button {
                 text: "Button with Icon";
                 icon: @image-url("../../thumbsup.png");
-                enabled: !GallerySettings.widgets-disabled;
+                enabled: GallerySettings.widgets-enabled;
             }
 
             Button {
                 checkable: true;
                 text: self.checked ? "ON" : "OFF";
-                enabled: !GallerySettings.widgets-disabled;
+                enabled: GallerySettings.widgets-enabled;
             }
         }
     }
 
     GroupBox {
-        title: "CheckBox - SpinBox - ComboBox";
+        title: "CheckBox - SpinBox - ComboBox - Switch";
         vertical-stretch: 0;
 
-        HorizontalBox {
+        HorizontalLayout {
             alignment: start;
+            spacing: 8px;
```

Comment:
I'm curious: Is there a specific reason to override the style's built-in spacing here?

Reply:
I uses here a `HorizontalLayout` and not the `HorizontalBox`. If I would use a `HorizontalBox` it would add extra padding.

Reply:
Would it be possible instead to use a `HorizontalBox` and set the `padding` to zero? I think that would be cleaner.